seed/seedwriter/seed20.go

```diff
@@ -346,6 +356,9 @@ func (tr *tree20) writeAssertions(db asserts.RODatabase, modelRefs []*asserts.Re
 		}
 	}
 
+	// TODO: assertions for components that are not in the model (but their snap
+	// is in the model) still end up here, rather than extra-snaps. to be more
+	// consistent, they should go in extra-snaps
```

Comment:
We could just stop writing the extra-snaps file? As far as I can tell, we don't use it for anything, and it doesn't contain any information that we can't re-construct from options.yaml, I believe.